### PR TITLE
Adding Mob Damages

### PR DESCRIPTION
**Pull request by ToufouMaster**

### CODE REVIEW

File: src/mobs/fight.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            entityId: mob.entity_id,
+            entityStatus: life > 0 ? 2 : 3, // Hurt Animation and Hurt Sound (sound not working)
+          })
+        }
+        return life
+      }, null)
+    )
+  }
+
+  /* if (newlife > 0) {
```
Comment (DeltaEvo) [inline src/mobs/fight.js:52]: Can you remove this comment ?

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

File: src/mobs/fight.js (hunk 1 above)
Comment (ToufouMaster) [reply src/mobs/fight.js:52]: sry

File: src/mobs.js [hunk 2]
```diff
@@ -24,6 +27,7 @@ export function register_mobs(world) {
       closed: [],
       start_time: 0,
       speed: 500 /* block/ms */,
+      life: 20 /* halfheart */,
```
Comment (DeltaEvo) [inline src/mobs.js:30]: I think `health` is more appropriate here

**Review by DeltaEvo — APPROVED** [review]

**Comment by ToufouMaster** [comment]
Et bah voila
xD

**Pull request merged**